_posts/2020-11-09-ten-tips-for-tidy-data.markdown

```diff
@@ -0,0 +1,66 @@
+
+* Use consistent variable names (“leaf_number” vs. “num_of_leaves”)
+* Use consistent codes for categorical variables (“female” vs. “F”)
+* Use a consistent code for any missing values (“NA” vs. “—” vs. a blank cell)
+
+## 3.	Date format
+Dates should be formatted using the global ISO 8601 standard which is YYYY-MM-DD. For example, 2020-11-06 is November 6th, 2020. This is standard and helps to avoid confusion between collaborators of different countries. Beware the automatic date formatting that happens in excel!
+
+## 4.	Choose good names
+In general, think carefully about how you name things. This applies to both file names and column names. They should be descriptive enough to avoid confusion, but concise enough to be functional. As a general rule, do not use spaces or other special characters in file or column names, except for underscores or hyphens. And of course, think about Tips #2 and #3 when naming things.
+
+## 5.	No empty cells
+It is generally accepted to use “NA” when a cell is meant to contain no value. This makes it clear that the cell was not left blank by mistake. And this will come in handy later, when sorting or manipulating data, to know the common code for a missing value.
+
+## 6.	Just one value per cell
+During data collection, there may be reason to store multiple, related data values in the same cell. Perhaps a code identifying an individual, that contains two pieces of information about that individual (like “adult_male_01”) or two related measurements (like “length x width”). But, in a tidy spreadsheet, these should be treated like two distinct pieces of data, and split into different columns accordingly. In the future, you may want to sort or compare your data by these different variables, and you’ll be glad you split them.
+
+## 7.	Make it a rectangle
+Your spreadsheet should be organized into a single rectangle with rows corresponding to subjects and columns corresponding to values. Often, data is not collected this way. But if you’re able to combine it into one rectangle, your computing and analysis steps will be much more streamlined.
+
+## 8.	Do not use font color or highlighting as data
+It may be tempting to utilize all of the color and visualization tools that programs like Excel offer to display information visually. For example, highlighting certain cells where the values are uncertain or using fonts to distinguish values in some way. But when you begin to analyze the data, it can be really difficult to extract that information for use in an analysis. Any important information should be stored in a way that is plain text readable. If you were to take away any color or font formatting, would all of that information still be clear?
```

Comment:
"It may be tempting to utilize all of the color and visualization tools" - all the color

Reply:
Thanks. I'll modify this phrase.